tests/acceptance/blocks/verticalresults.js

```diff
@@ -66,6 +69,19 @@ class VerticalResults {
     return this._selector.exists;
   }
 
+  /**
+   * wait for results to return based on search complete state 
+   * (default selector timeout of 10 seconds)
+   */
+   async waitOnSearchComplete () {
```

Comment:
If someone calls this method, but no search has actually been executed, will it timeout? I'm wondering if we should handle that situation gracefully, like it returns immediately. Maybe not something we need to worry about, just a thought.

Reply:
in this method, and in the updated one, it will timeout eventually. I think if no search is executed and someone uses this method, meaning they are expecting a search or result returned of some sort, it make sense to indicate a failure here to notify that the expected behavior is not met